scripts/speech_recognition/k2/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+K2_MAKE_ARGS="-j" pip install git+https://github.com/k2-fsa/k2@v1.11#egg=k2
```

Comment:
What is the installation time here? And is it possible to unpin the version? Or fixed version is necessary for the time being?

Reply:
The installation time is ~2 min on my workstation and ~15 min on CI. It's _relatively_ safe to use the latest version, so I modified the script in [8d3e482](https://github.com/NVIDIA/NeMo/pull/3351/commits/8d3e48215629824b59a020526a0444c944b2d1c2) to install the latest release.

Reply:
15 minutes on CI is not at all small. The current CI installation step is roughly 4 minutes total, adding 15 minutes to that is not possible. We will have to add this to the docker file and instead run nightly tests.